windows/src/desktop/kmshell/main/Keyman.System.UpdateStateMachine.pas

```diff
@@ -1085,9 +1091,10 @@ procedure InstallingState.HandleCheck;
 
 function InstallingState.HandleKmShell;
 begin
-  // Result = exit straight away as we are installing (MSI installer)
-  // need to just do a no-op keyman will it maybe using kmshell to install
-  // packages.
+  // Should not be called while in installing state MSI installer may have
```

Comment:
Is there a period missing after "state"? I have a hard time parsing the sentence as it is.